qcodes/dist/schemas/station.schema.json

```diff
@@ -0,0 +1,176 @@
+{
+    "type": "object",
+    "title": "QCoDeS Station configuration validation",
+    "description": "This schema serves the purpose of validating yaml files that describe a QCoDeS station. See more here: https://qcodes.github.io/Qcodes/examples/Station.html#Configuring-the-Station-by-using-a-YAML-configuration-file",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "definitions": {
+        "instruments": {
+            "enum": [
+                ""
+            ]
+        },
+        "parameter_base": {
+            "$comment": "A common base for the properties shared by 'add_parameters' and 'parameters can first be implemented with the new json-schema draft 2019-09, for which there is no implementation at this point. The new feature is called 'unevaluatedProperties'."
+        }
+    },
+    "properties": {
+        "instruments": {
+            "description": "Instruments defined in this file. Valid names are all alphanumerical names including '-' and '_'.",
```

Comment:
hyphens cannot be part of a python identifier. or am i misunderstanding something?

Reply:
Thank you for putting it so gallantly. To my shame I did not know that hyphens where not allowed in python identifiers.